case/infrasim/virtual_node/T130052_idic_ESXiTest.py

```diff
@@ -37,7 +37,15 @@ def boot_to_disk(self, node):
 
         dst_path = node.send_file(os.environ["HOME"]+"/images/esxi6p3-1.qcow2", "/tmp/esxi6p3-1.qcow2")
         #dst_path = "/tmp/esxi6p3-1.qcow2"
-
+        rsp = node.ssh.send_command_wait_string(str_command=r"md5sum /tmp/esxi6p3-1.qcow2"
+                                               +chr(13),wait="~$")
+        if  "310579038c772d8222f1f0f2cdfbff35" in rsp:
```

Comment:
@Summer-Qing , though it might not be a big deal, but using const with a human-readable name is a good practice, have a try~